src/FBLoginManager.js

```diff
@@ -61,11 +61,26 @@ export type LoginResult = {
   declinedPermissions?: Array<string>,
 };
 
+export type LoginTracking = 'enabled' | 'limited';
+
 module.exports = {
   /**
-   * Logs the user in with the requested permissions.
+   * Log in with the requested permissions.
+   * @param loginTracking IOS only: loginTracking: 'enabled' | 'limited', default 'enabled'.
+   * @param nonce IOS only: Nonce that the configuration was created with. A unique nonce will be used if none is provided to the factory method.
    */
-  logInWithPermissions(permissions: Array<string>): Promise<LoginResult> {
+  logInWithPermissions(
+    permissions: Array<string>,
+    loginTracking?: LoginTracking,
```

Comment:
I see it is not standard in this lib, but other libs I work with also append the platform to these arguments to make it more clear that they are not used for all platforms. So similar as was done for the properties for FBLoginButton. But, as I'm not the maintainer here 🤷🏻‍♀️

Reply:
I totally agree with your point here :)

Reply:
What will be best practice for this?
Create **_logInWithPermissionsIOS_** and **_logInWithPermissionsAndroid_**

or something like this 
`logInWithPermissions ( { permissions,  loginTrackingIOS, nonceIOS } ) `

or did you mean
```
logInWithPermissions(
    permissions: Array<string>,
    loginTrackingIOS?: LoginTracking,
    nonceIOS?: string,
  )
```

Reply:
@cocoa-rum

I mean the latter. So:

```
logInWithPermissions(
    permissions: Array<string>,
    loginTrackingIOS?: LoginTracking,
    nonceIOS?: string,
  )
```